cdap-ui/app/cdap/components/PipelineList/DraftPipelineView/store/ActionCreator.ts

```diff
@@ -43,16 +45,29 @@ function getOldDrafts(oldDrafts) {
 
 export function getDrafts() {
   Observable.forkJoin<
-    IDraft[],
+    any,
     any /** setting as 'any'. We should remove user store reference in the next major release */
   >(
     MyPipelineApi.getDrafts({
       context: 'default',
-    }),
+    })
+      .map((res) => res)
```

Comment:
What does this `.map` do?

Reply:
Doesn't seem to require. I was under the assumption that the map is required for catch'ing the errors. Have removed it.